CONTRIBUTING.md

```diff
@@ -108,6 +108,8 @@ It is encouraged to write all new docstrings in Markdown markup.  If you need to
 Many of the existing docstrings are currently restructured text codeblocks and these will be transitioned to Markdown over time.  RST codeblocks are delineated with the triple-quote (\`\`\`rst  \`\`\`) Makdown codeblock syntax.
 The content of the codeblock is spliced directly into the final restructured text document unmodified.
 
+If you decide to add some news to `NEWS.md` and you include a reference to a pull request or issue you should make sure to run `julia doc/NEWS-update.jl` from Julia's root directory before you submit. Running this ensures that the cross-reference links are up-to-date.
```

Comment:
Shouldn't you delete the old wording then?  It looks like you have both paragraphs in the PR.